### DISCUSSION TITLE
运行的时候一直显示DoLog4j:  false true

### DISCUSSION THREAD

**Discussion by purple-WL** [hktalent/scan4all#23] [Q&A]
<img width="687" alt="image" src="https://user-images.githubusercontent.com/63894044/180184199-591ac87a-cc0e-48b8-b9d4-3d8603be80dc.png">

**Comment by hktalent** [comment]
@purple-WL sorry,Forgot to block debug code

**Reply by hktalent** [reply]
mkdir ~/MyWork/;cd ~/MyWork/;git clone  https://github.com/hktalent/log4j-scan

**Reply by hktalent** [reply]
Will be fixed in the next version tonight at 21:00

**Comment by purple-WL** [comment]
badger 2022/07/21 18:06:26 INFO: Got compaction priority: {level:0 score:1.73 dropPrefixes:[]}

**Reply by hktalent** [reply]
> badger 2022/07/21 18:06:26 INFO: Got compaction priority: {level:0 score:1.73 dropPrefixes:[]}

Will be fixed in the next version tonight at 21:00

thanks
